styles/custom.scss

```diff
@@ -8,54 +8,51 @@ $link-color: #0d3e79;
 $link-decoration: none;
 $hr-color: #dee2e6;
 $hr-opacity: 1;
-$link-decoration: none;
 $nav-shadow: #d5d5d5;
 
+.link {
+  border-bottom: 1px dotted;
```

Comment:
links will now by styled with dotted underlines so i made a class for them here. i leave the `$link-decoration` as none to start us at a blank slate otherwise there would be single underlines under all the links on top of the dotted underlines.